ratatool-scalacheck/src/main/scala/com/spotify/ratatool/scalacheck/package.scala

```diff
@@ -165,4 +168,19 @@ package object scalacheck extends AvroGeneratorOps
     }
   }
 
-}
+  implicit def arbSpecificRec[T <: SpecificRecord : ClassTag]: Arbitrary[T] = {
```

Comment:
do we want to document these in a README?

Reply:
👍 Added a note